modules/localgov_services_status/localgov_services_status.module

```diff
@@ -7,6 +7,7 @@
 
 use Drupal\Core\Cache\Cache;
 use Drupal\Core\Url;
+use Drupal\localgov_roles\RolesHelper;
```

Comment:
Will this use statement work if localgov_roles is not installed and enabled?
(Assume the code will exist becuase localgov_core is a dependancy).